code/mathematical-algorithms/convert_to_roman/convertToRoman.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Please add open genus comment at the top ....please refer to the community guidelines for the same ! 👍

Reply:
Can you check if everything is ok now please?